### PR TITLE
[Merged by Bors] - add more tests for the front office and new asserts

### PR DESCRIPTION
**Pull request by vadpopov**
- more tests for the front office
- more asserts
- clean up

### CODE REVIEW

File: discovery_engine_api_tests/tests/back_office/test_actions_with_documents.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class TestActionsWithDocuments:
+    api_handler = ApiHandler()
+
+    @allure.severity(allure.severity_level.NORMAL)
+    def test_delete_doc(self, ingest_generated_document):
+        request = self.api_handler.delete_document(doc_id=ingest_generated_document)
+        au.assert_status_code_equals(request.status_code, 204)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: create ticket")
```
Comment (janpetschexain) [inline discovery_engine_api_tests/tests/back_office/test_actions_with_documents.py:21]: 🔧 ticket already exists
```suggestion
    @pytest.mark.skip(reason="ET-4004")
```

File: discovery_engine_api_tests/tests/front_office/test_interactions.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    @allure.severity(allure.severity_level.CRITICAL)
+    def test_positive_interaction(self, ingest_generated_document):
+        user_id = tu.generate_random_alphanumerical(6)
+        positive_interaction = Interactions(Interaction(id=ingest_generated_document, type="Positive")).to_json()
+        request = self.api_handler.interact_with_documents(user_id, positive_interaction)
+        au.assert_status_code_equals(request.status_code, 204)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: need to be updated")
```
Comment (janpetschexain) [inline discovery_engine_api_tests/tests/front_office/test_interactions.py:23]: ❓ does this refer to code in the engine itself, if so is there already a ticket for that? or does this refer to the id and interaction constructors in the test?

File: discovery_engine_api_tests/tests/front_office/test_personalization.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    def test_invalid_user_id(self):
+        user_id = su.generate_invalid_id()
+        request_docs = self.api_handler.get_personalized_docs(user_id)
+        au.assert_status_code_equals(request_docs.status_code, 400)
+
+        documents_response = self.api_handler.deserialize_json(request_docs.text)
+        au.assert_strings_equal(documents_response["kind"], "InvalidUserId")
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: need to be updated")
```
Comment (janpetschexain) [inline discovery_engine_api_tests/tests/front_office/test_personalization.py:32]: ❓ does this refer to code in the engine itself, if so is there already a ticket for that?

File: discovery_engine_api_tests/tests/front_office/test_interactions.py (hunk 2 above)
Comment (vadpopov) [reply discovery_engine_api_tests/tests/front_office/test_interactions.py:23]: @janpetschexain DMed you with explanation

**Review by janpetschexain — APPROVED** [review]
lgtm

**Comment by vadpopov** [comment]
bors merge

**Pull request closed without merging**